tests/test_ingest.py

```diff
@@ -114,11 +114,19 @@ def test_ingest_dense_matrix(self):
         """Ingest Pipeline should extract and transform dense matrices
         """
 
-        args = (
-            'ingest_expression '
-            '--matrix-file gs://fake-bucket/tests/data/dense_matrix_19_genes_100k_cells.txt '
-            '--matrix-file-type dense'
-        )
+        args = [
```

Comment:
Kudos again on the elegant solution for supporting JSON argument values by making `args` a list.